SuperNova/README

```diff
@@ -0,0 +1,4 @@
+Convert the ipynb file to python script before lint:
+
+$ jupyter nbconvert SuperNova.ipynb --stdout --to python --PythonExporter.exclude_markdown=True|flake8 - --max-line-length 120 --ignore=F841,W391
```

Comment:
We prefer to stick to 80 max line length. Where does it exceed it? In the notebook?

Reply:
many lines have more than 100 characters, I tried to fix it,  but the fixed version looks super ugly, do you mind?

Reply:
https://pep8.org/#maximum-line-length
We, as a group, have resisted the urge to increase this limit. With some exceptions, there are ways to make the code look nice with the given 80 characters: use temporary variables, break chained commands, etc. If the standard library adheres to this we surely can too.